benchmarks/yadmark/data/observables.py

```diff
@@ -28,10 +28,6 @@
 default_config = {
     0: {"observable_names": ["F2light"], "kinematics": default_kinematics},
     1: {
-        "observable_names": ["F2light", "F2total", "FLtotal", "F3total"],
-        "kinematics": default_kinematics,
-    },
-    2: {
         "observable_names": ["F2light", "FLlight", "F3light"],
```

Comment:
This is not a good idea - the default should still be total, since this is available for everybody in contrast to light